slices/ovn-central.yaml

```diff
@@ -0,0 +1,10 @@
+package: ovn-central
+
+slices:
+  base:
+    essential:
+      - openvswitch-common_bins
+      - ovn-common_bins
+    contents:
+      /usr/bin/ovn-northd:
+      /usr/share/ovn/*.ovsschema:
```

Comment:
Are these config files? If so, perhaps it's better to create a `config` slice for these and include it as an `essential`. I think `/etc/default/ovn-central` would probably also deserve to be there.

Reply:
Done